### PR TITLE
### Milestone: Portfolio details popup window

### PR DESCRIPTION
**Pull request by ibrahim777764**
### Milestone: Portfolio details popup window

- Use basic JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- understood how to parse a Figma design to create a UI.
- Use objects to store and access data.

### CODE REVIEW

**Comment by ibrahim777764** [comment]
please review my code,thankyou

File: index.js [hunk 1]
```diff
@@ -1,31 +1,147 @@
...
+  // //   grid.appendChild(clonedNode);
+
+// //   showProjectButton.addEventListener('click', () => {
+// //     detailsModal.classList.remove('hidden');
+// //     const paragraghDescription = detailsModal.querySelector('p');
+// //     const detailsHeader = detailsModal.querySelector('h3.details-heading');
+// //     paragraghDescription.textContent = project.description;
+// //     detailsPopUpHeading.textContent =  project.name;
+// //   });
+// });
```
Comment (gemmen29) [inline index.js]: Kindly get rid of the comments in order to make your code cleaner.

File: index.js [hunk 2]
```diff
@@ -1,31 +1,147 @@
...
+
+    detailsPopUpHeading.textContent = project.name;
+
+    const btnPopups = recentWorks.querySelectorAll('.btn-popup');
+    btnPopups.forEach((button) => {
+      button.addEventListener('click', () => {
+        popupWindow.classList.remove('hidden');
+      });
+    });
+  });
```
Comment (gemmen29) [inline index.js:116]: Kindly try to match the `see live` and `see source` in the desktop version width `992px` to match the Figma design. 
| Your design | Figma design |
| --- | --- |
| image | image |
the text and icon should be beside each other.

Comment (gemmen29) [inline index.js:116]: Kindly add the scroll for the details modal in mobile version width `375px` in order all your modal content appear.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @ibrahim777764, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibrahim777764** [comment]
completed all required changes,hope so every thing is ok now ,thankyou

File: index.html [hunk 3]
```diff
@@ -24,6 +24,60 @@
...
+                    **heading**
+                </h2>
+                <div class="popup-button-group-desktop">
+                    <button type="button" class="btn-popup">
+                        See Live
+                        <img alt="shortcut" class="icon-btn" src="./exp.png" />
+                    </button>
+                    <button type="button" class="btn-popup">
+                        See Source
+                        <img alt="icon" class="icon-btn" src="gitexp.png" />
```
Comment (ShoiraTa) [inline index.html:44]: - [ ] Please make sure that in mobile view the content inside the buttons does not wrap: 

Screenshot 2022-03-10 at 13 48 32
 You can achieve that by decreasing the font size, deleting the letter spacing, and decreasing the padding. 
Try this: 

```
.popup-button-group > .btn-popup {
    font-size: 15px;
    letter-spacing: 0;
    justify-content: center;
    align-items: center;
}
```

File: index.html [hunk 4]
```diff
@@ -358,6 +328,60 @@ <h2 class="footer-title">
         </ul>
         <a href="https://mail.google.com/" class="link-mail">ibrahimnaseer896.gmail.com</a>
     </footer>
+ 
+<!-- <div id="myModal" class="modal">
+
+  
+  <div class="modal-content">
```
Comment (ShoiraTa) [inline index.html]: - [ ] OPTIONAL:  I can see lots of commented code here, if you do not need it anymore lets delete it and make sure that the project is clean and professional

File: index.js [hunk 5]
```diff
@@ -1,31 +1,117 @@
...
```
Comment (ShoiraTa) [inline index.js:116]: - [ ] As the previous reviewer mentioned, make sure that the text and icon are beside each other. They are still wrapping: 
Screenshot 2022-03-10 at 13 54 32

Try this:
```
button {
font-size: 15px;
letter-spacing: 0px;
}
```

**Review by ShoiraTa — CHANGES_REQUESTED** [review]
Hi @ibrahim777764 ,

:thumbsup: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are some issues that you still need to work on to go to the next project but you are almost there!



 ## Required Changes  :recycle: 

_Check the comments under the review._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear by mentioning my Github username.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibrahim777764** [comment]
please review my code , thankyou

**Review by elyor-doniyorov — APPROVED** [review]
Hi @ibrahim777764, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**